static/js/main.js

```diff
@@ -13505,3 +13505,18 @@ $(document).ready(function() {
 $(document).ready(function() {
     $('.alert').createAlert();
 });
+
+/* ==========================================================================
+   Footer Button: Scroll to Top
+   ========================================================================== */
+
+$(document).ready(function() {
+    var duration = 300;
+
+    $('.footer_top-button').click(function(event) {
+        jQuery('html, body').animate({scrollTop: 0}, duration);
+        return false;
```

Comment:
Since you're capturing the `event` you could do `event.preventDefault();` immediately within the event (or even shorten it to to `e`) instead of `return false;`